### DISCUSSION TITLE
Request update RPC URLs and icons for Klaytn Blockchain

### DISCUSSION THREAD

**Discussion by dylanngph** [family/connectkit#303] [General]
Hi, i am dev ambassador of Klaytn Foundation, i just find out that connectkit currently doesn't support Klaytn icons and the rpc urls is old. Please support this. Wait for your response.

**Comment by lochie** [comment]
The data for `rpcUrls` is inherited from the [`wagmi`](https://github.com/wagmi-dev/wagmi) peer dependency, you'll need to request the update in their repo.
See reference: https://github.com/wagmi-dev/viem/blob/main/src/chains/definitions/klaytn.ts

**Reply by dylanngph** [reply]
Yes, about hte rpcUrl we can create custom one, but what about the chain icon?
